### ISSUE TITLE
Integrate repo with transifex

### ISSUE THREAD

**Issue by marksweb** [django/djangoproject.com#1759]
To make the process of translating the website smoother, integration between this repo & transifex would be a great help.

But I believe it needs someone with maintainer/admin on transifex and admin on here (settings access)

Anyone with the relevant permissions can then set it up from here;
https://app.transifex.com/django/djangoproject-com/settings/integrations/

**Comment by bmispelon** [comment]
@sarahboyce I thought I read somewhere recently that you were an admin on Transifex. If I'm reading it right, you have admin access to this repo also, so you should be able to do what Mark asks for I think 🤔 

(As far as I can find, ops has no access to Transifex)

**Comment by marksweb** [comment]
@bmispelon Yes, it seems like fellows & Carlton have the transifex access. So I think Sarah could assist here - please let me know if there are any uncertainties.

**Comment by ulgens** [comment]
Any chance to revisit this issue?

**Comment by sarahboyce** [comment]
Looks like I have admin access of django but not djangoproject.com. Perhaps @claudep can help here

**Comment by claudep** [comment]
I don't have access to any django/djangoproject.com admin.

Could you explain what integration means for you? You know that djangoproject.com is already translatable on Transifex, right?

**Comment by marksweb** [comment]
I have transifex access, but we need someone with access to transifex and github in order to authorise the integration and add the app to github. 

I think there's some outstanding PRs of mine adding translations.

Then there needs to (perhaps) be some sort of ops decision on fitting translation into workflows.

And then of course the community needs to do some translating!

**Comment by claudep** [comment]
Thanks for the reminders.

If we want to follow the workflow used for docs, some management command (https://github.com/django/djangoproject.com/blob/main/docs/management/commands/update_docs.py is used for docs, putting pot files to https://docs.djangoproject.com/pots/<docs_domain>.pot) should produce the pot file(s), and then we can configure Transifex to use those files as the source.
